docs/manifest/moreinfo.md

```diff
@@ -35,9 +35,19 @@ None.
 
 | Element | Required | Description |
 | ------- | ------- | -------|
-| **MoreInfoText** | Yes | Specifies additional information in the preprocessing dialog of a spam-reporting add-in to help users report unsolicited messages. Its **resid** attribute must be set to the value of the **id** attribute of a [String](string.md) in the [ShortStrings](shortstrings.md) element under the [Resources](resources.md) element. |
+| **MoreInfoText** | Yes | Specifies additional information in the preprocessing dialog of a spam-reporting add-in to help users report unsolicited messages. The **resid** attribute must be set to the value of the **id** attribute of a [String](string.md) in the [ShortStrings](shortstrings.md) element under the [Resources](resources.md) element. Depending on the Outlook client, the text specified in the **\<MoreInfoText\>** element appears before the URL that's provided in the **\<MoreInfoUrl\>** element or as link text for the URL. For more information, see [MoreInfoText](#moreinfotext).|
 | **MoreInfoUrl** | Yes | Specifies the URL of a site containing informational resources in the preprocessing dialog of a spam-reporting add-in. Its **resid** attribute must be set to the value of the **id** attribute of a [Url](url.md) in the [Urls](urls.md) element under the [Resources](resources.md) element. |
 
+### MoreInfoText
+
+In Outlook on the web, classic Outlook on Windows (starting in Version 2404 (Build 17526.15020)), and [new Outlook on Windows](https://support.microsoft.com/office/656bb8d9-5a60-49b2-a98b-ba7822bc7627), the **\<MoreInfoText\>** element specifies the custom link text of the URL that's provided in the **\<MoreInfoUrl\>** element. On these Outlook clients, the link is automatically prefixed with the static string, "For more info go to: ".
```

Comment:
I know this is the actual text, but seems like it should say "For more info, go to:"

Reply:
Yes. I had provided the feedback to the crew.